ebuild-writing/user-submitted/text.xml

```diff
@@ -29,14 +29,15 @@ commit message format section</uri>, is highly encouraged.
 Note that ebuilds received in the form of git patches or pull requests
 will have the user as the commit author by default, in which case
 including the user information in the commit message explicitly
-may not be necessary.
+may not be necessary. If that is not the case, you may wish to use
+git commit's <c>--author</c> parameter to explicitly give them credit.
```

Comment:
This is problematic. If the patch doesn't originate from `git format-patch` (i.e. doesn't have the correct author), then chances are that it will also miss a `Signed-off-by:` line and therfore violate our copyright policy.

So I'd rather discourage using the --author option. Any nontrivial patches should be submitted with format-patch, and trivial (= not copyrightable) suggestions can be tagged with `Suggested-by:` or `Reported-by:` as the section already says.

Reply:
I don't think that's likely, actually.. you should obviously be checking for our ```Signed-off-by``` either way, and often, contributors on Bugzilla will attach an ebuild and post the sign-off in a comment when requested.

@gktrk was generally a fan of this suggestion in the past. Of course, ```format-patch``` is preferred, but not always doable by users.

Reply:
> [...] you should obviously be checking for our `Signed-off-by` either way, [...]

Mention that then?

Reply:
Good plan.